timber-lint/src/test/java/timber/lint/WrongTimberUsageDetectorTest.kt

```diff
@@ -184,6 +188,8 @@ class WrongTimberUsageDetectorTest {
                 |  }
                 |}""".trimMargin())
         )
+        .allowMissingSdk()
+        .allowCompilationErrors()
```

Comment:
this is for debugging and shouldn't really land on master

Reply:
I think the issue was that `import static java.lang.String.format;` causes a compilation error in Kotlin

Reply:
Ah, got it.  Confirmed that it's a Lint bug that's already been fixed in AGP 7.1 alpha: https://groups.google.com/g/lint-dev/c/BigCO8sMhKU